wandb/apis/public.py

```diff
@@ -1487,12 +1487,13 @@ def used_artifacts(self, per_page=100):
         return RunArtifacts(self.client, self, mode="used", per_page=per_page)
 
     @normalize_exceptions
-    def use_artifact(self, artifact):
+    def use_artifact(self, artifact, use_as=None):
         """Declare an artifact as an input to a run.
 
         Arguments:
             artifact (`Artifact`): An artifact returned from
                 `wandb.Api().artifact(name)`
+            use_as (string, optional): A string identifying how the artifact is used.
```

Comment:
Would be good to give more context here to the user.  I have no idea what this means.

Reply:
Thats a great point.